spec/support/shared_contexts/rake.rb

```diff
@@ -0,0 +1,19 @@
+require 'rake'
+
+shared_context 'rake' do
+  let(:rake)      { Rake::Application.new }
```

Comment:
Do you actually want to follow this rule, or should the Hound config be updated to allow columnar whitespace?